src/css/_search.scss

```diff
@@ -1,20 +1,24 @@
+$width: 250px;
+
 div.choices {
   position: absolute;
   top: 5.4em;
   left: 3em;
-  max-width: 250px;
+  max-width: $width;
   z-index: 1000;
 
   @media only screen and (min-width: 48em) {
     top: 7em;
   }
 
+  // Changes the search list.
   &.is-open {
-    width: 250px;
```

Comment:
Is this necessary? I thought the point is that regardless of whether it's open, it will have the same size.

Reply:
Yeah, the way choices.js is written, we have to set this value.